tide_site.module

```diff
@@ -234,6 +236,56 @@ function tide_site_views_pre_view(ViewExecutable $view, $display_id, array &$arg
     ];
     $view->addHandler($display_id, 'field', 'media__field_media_site', 'field_media_site', $site_field, 'field_media_site');
   }
+  if ($view->id() == 'content' && $view->current_display == 'page_1') {
+    if (_has_target_site_fields()) {
+      $fields_order = [
+        'title',
+        'field_node_primary_site',
+        'field_node_site',
+        'name',
+        'type',
+        'moderation_state',
+        'changed',
+        'operations',
+      ];
+      $filters_order = [
+        'title',
+        'status_extra',
+        'field_node_primary_site_target_id',
+        'field_node_site_target_id',
+        'uid',
+        'type',
+        'moderation_state',
+        'status',
+      ];
+      $module_handler = Drupal::moduleHandler();
+      $configs = Yaml::parseFile($module_handler->getModule('tide_site')->getPath() . '/yaml/content_summary_page.yml');
```

Comment:
@vincent-gao  @anthony-malkoun @vincent-gao To be honest, I don't like the idea of loading a custom YAML and replace individual view components with components from the loaded YAML. If we decide to change the `views.view.content` that much, we should create our own custom view, and alter the `system.admin_content` route to our view. Ideally, there should be a setting to toggle the view replacement.

Reply:
@anthony-malkoun  @sonnykt  I will take this advice.